ocs_ci/ocs/resources/pvc.py

```diff
@@ -308,6 +308,28 @@ def get_all_pvcs_in_storageclass(storage_class):
     return out
 
 
+def get_all_pvs_in_storageclass(storage_class):
```

Comment:
Please consider a simpler option.
To get all PVCs in a namespace the following oc command is good enough. 
`oc get pvc -n openshift-storage -o template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}'`

Reply:
This function is to get the PVs not the PVCs

Reply:
Yes, it works the same way for PVs too. I just gave an example of PVCs.